fiftyone/server/routes/samples.py

```
@@ -0,0 +1,283 @@
+
+
+async def _resolve_dataset_name(dataset_id: str) -> Optional[str]:
+    """Resolve a dataset id to its name, or ``None`` if it does not exist."""
+    db = foo.get_async_db_conn()
+    ds = await db["datasets"].find_one(
+        {"_id": ObjectId(dataset_id)}, {"name": 1}
+    )
+    return ds["name"] if ds is not None else None
+
+
+async def _build_item(
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_build_item`

(ANN202)